### PR TITLE
Rename collapsed naming convention to pinned and small code clean up

### PR DESCRIPTION
**Pull request by bernardwang**

### CODE REVIEW

File: src/engine-scripts/puppet/onReady.js [hunk 1]
```diff
@@ -25,26 +25,24 @@ module.exports = async ( page, scenario ) => {
...
-		await require( './collapsedTocState' )( page, hashtags );
-	}
+		await require( './unpinnedTocState' )( page, hashtags );
 
-	if ( hashtags.includes( '#toggle-toc-subsections' ) ) {
-		await require( './toggleTocSubsections' )( page );
+		if ( hashtags.includes( '#toggle-toc-subsections' ) ) {
+			await require( './toggleTocSubsections' )( page );
+		}
+		if ( hashtags.includes( '#echo' ) ) {
```
Comment (nicholasray) [inline src/engine-scripts/puppet/onReady.js:33]: The [echo report](https://pixel.wmcloud.org/reports/echo/index.html) tests both the vector-2022 and minerva skin so we should probably keep this outside the conditional

**Review by nicholasray — COMMENTED** [review]
Thanks for this PR and sorry for the delay of  my review. I just had one comment below

File: src/engine-scripts/puppet/onReady.js (hunk 1 above)
Comment (nicholasray) [reply src/engine-scripts/puppet/onReady.js:33]: Also, CI is failing because of some eslint issues

**Comment by bernardwang** [comment]
Closing in favor of a more comprehensive pinnable element and dropdown solution

**Pull request closed without merging**